components/main-chef-wrapper/platform-lib/version_windows.go

```diff
@@ -147,3 +148,45 @@ func ExpectedOmnibusRoot() string {
 	rootPath = strings.Replace(s, "//", "/", -1)
 	return rootPath
 }
+
+func UnmarshallRubyEnv() map[string]interface{} {
```

Comment:
i think we should keep one func to unmarshal data it's not platform specific.read file then send byte to unmarshal.

Reply:
Yes, we can do that. I had done that initially, but later when i realised path sperator were different for windows and unix i shifted it back.

Reply:
so i moved it to each platform so that each platform uses it's own unmarshall, based on path it has.

Reply:
ok, but we can pass file path and then we can use, but it's ok